fastify.d.ts

```diff
@@ -458,7 +479,7 @@ declare namespace fastify {
      * Extends the standard server error. Return an object with the properties you'd
      * like added to the error
      */
-    extendServerError(extendFn: (error: Error) => Object): FastifyInstance<HttpServer, HttpRequest, HttpResponse>
+    extendServerError(extendFn: (error: FastifyError) => Object): FastifyInstance<HttpServer, HttpRequest, HttpResponse>
```

Comment:
should this be a FastifyError?

Reply:
As far as I understand this method will extend error that come from Fastify. So if there will be some validation error (with validation field), then it should be `FastifyError`, because if not, there will be no access to `error.validation` in ts. It will throw an error during compile unless force type casting will be used.

Reply:
Ok